Formula/pumba.rb

```diff
@@ -24,5 +24,7 @@ def install
   test do
     output = shell_output("#{bin}/pumba --version")
     assert_match "Pumba version 0.4.4", output
+    output = pipe_output("#{bin}/pumba rm test-container 2>&1")
```

Comment:
This is current behavior, I will consider to change it in next release and will add a check to validate error code.